snow/engine/snowman/transitive.go

```diff
@@ -259,30 +260,43 @@ func (t *Transitive) PushQuery(ctx context.Context, nodeID ids.NodeID, requestID
 	return t.buildBlocks(ctx)
 }
 
-func (t *Transitive) Chits(ctx context.Context, nodeID ids.NodeID, requestID uint32, blkID ids.ID, acceptedID ids.ID) error {
+func (t *Transitive) Chits(ctx context.Context, nodeID ids.NodeID, requestID uint32, preferredID ids.ID, preferredIDAtHeight ids.ID, acceptedID ids.ID) error {
 	t.acceptedFrontiers.SetLastAccepted(nodeID, acceptedID)
 
 	t.Ctx.Log.Verbo("called Chits for the block",
-		zap.Stringer("blkID", blkID),
 		zap.Stringer("nodeID", nodeID),
-		zap.Uint32("requestID", requestID))
+		zap.Uint32("requestID", requestID),
+		zap.Stringer("preferredID", preferredID),
+		zap.Stringer("preferredIDAtHeight", preferredIDAtHeight),
+		zap.Stringer("acceptedID", acceptedID),
+	)
 
-	added, err := t.issueFromByID(ctx, nodeID, blkID)
+	addedPreferred, err := t.issueFromByID(ctx, nodeID, preferredID)
 	if err != nil {
 		return err
 	}
 
-	// Will record chits once [blkID] has been issued into consensus
+	addedPreferredIDAtHeight, err := t.issueFromByID(ctx, nodeID, preferredIDAtHeight)
```

Comment:
Should we just set `addedPreferredIDAtHeight` to true, if `preferredID == preferredIDAtHeight `?